Create a guide for FAIR software publishing (#346)
ubvu/open-handbook: guides/publishing-fair-software.qmd

---
title: "How can you publish FAIR software"

> image: '/public/cover-images/DALLE_DMP.jpg'

Reviewer: ```suggestion
image: '/public/RSM_sticker.png'
```

Well done! Your research code is not sitting on your own computer (or worse – deleted after publication*), but is available online for others to find. This is step one in following the [FAIR principles](../topics/fair-principles.qmd). In the remainder of this guide, you will find steps that will guide you through all steps to release a "perfect" code repository. The first sections are most important, further sections provide room for further improvement, so please use this guide at your own discretion.

*VU Amsterdam requires research data and software to be preserved for at least 10 years, unless legal provisions or discipline-specific guidelines dictate otherwise, as per the [Research Data and Software Management Policy](../topics/research-data-and-software-management-policy.qmd). Note that funding organisations may have specific requirements for preserving code as well. If you receive external funding for your research, please make sure to familiarise yourself with such requirements.

## Licence

```{.include shift-heading-level-by=4}
../topics/software-licensing.qmd
```

> ## DOI

Reviewer: ```suggestion
## Persistent Identifier
```

> A DOI, or Digital Object Identifier, string assigned to a document, dataset, or object to provide a permanent link. DOIs ensure that even if the URL of the content changes, the DOI link remains constant, making it a reliable way to cite and access digital materials. There are two ways to do this. Depending on how often you intend to update your code, one or the other may be simpler.

Reviewer: ```suggestion
A [persistent identifier](../topics/persistent-identifier.qmd)  is a durable reference to a digital dataset, document, website or other object. A Digital Object Identifier (DOI) is a widely used Persistent Identifier in the research domain and hence for research software as well. There are two ways to generate a DOI for your code. Depending on how often you intend to update your code, one or the other may be simpler.
```
